CodingDojo/server/src/main/java/com/codenjoy/dojo/services/TeamService.java

```diff
@@ -0,0 +1,80 @@
+package com.codenjoy.dojo.services;
+
+/*-
+ * #%L
+ * Codenjoy - it's a dojo-like platform from developers to developers.
+ * %%
+ * Copyright (C) 2018 - 2021 Codenjoy
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import com.codenjoy.dojo.web.rest.pojo.PTeam;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.mapping;
+import static java.util.stream.Collectors.toList;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class TeamService {
+
+    private final PlayerGames playerGames;
+
+    public List<PTeam> getTeamInfo() {
+        List<PTeam> teams = new ArrayList<>();
+        Map<Integer, List<PlayerGame>> playersByTeam = playerGames.all().stream()
```

Comment:
Тут бы порефачить, уверен есть боле лаконичный способ сделать это стримами

Reply:
It could be more conciseness but I'm not sure it's easier to read
e.g. 
![image](https://user-images.githubusercontent.com/43476908/125596330-9fe56a29-be7f-4263-93af-b0f360616442.png)
What would you say?